code/procs/mobprocs/chatprocs.dm

```diff
@@ -426,6 +428,40 @@
 	if (src.get_brain_damage() >= 60)
 		speechverb = pick("says","stutters","mumbles","slurs")
 
+	if (src.singing || (src.bioHolder && src.bioHolder.HasEffect("accent_elvis")))
+		// use note icons instead of normal quotes
+		var/note_type = src.singing == "bad" ? "notebad" : "note"
+		var/note_img = "<img class=\"icon misc\" style=\"position: relative; bottom: -3px; \" src=\"[resource("images/radio_icons/[note_type].png")]\">"
+		if (singing == "loud" || singing == "bad" || loudness > 0)
+			first_quote = "[note_img][note_img]"
+			second_quote = first_quote
+		else
+			first_quote = note_img
+			second_quote = note_img
+		// select appropriate singing verb
+		if (src.bioHolder && src.bioHolder.HasEffect("smoker"))
+			speechverb = "rasps"
+			if (singing == "loud" || loudness > 0)
+				speechverb = "sings Tom Waits style"
+		else if (src.traitHolder && src.traitHolder.hasTrait("french"))
+			speechverb = "sings [pick("Charles Trenet", "Serge Gainsborough", "Edith Piaf")] style"
+		else if (src.bioHolder && src.bioHolder.HasEffect("accent_swedish"))
+			speechverb = "sings disco style"
+		else if (src.bioHolder && src.bioHolder.HasEffect("accent_scots"))
+			speechverb = pick("laments", "softly sings", "croons", "sorrowfully intones", "sobs", "bemoans")
+		else if (src.bioHolder && src.bioHolder.HasEffect("accent_chav"))
+			speechverb = "grimily raps"
+		else if (src.singing == "bad")
+			speechverb = pick("makes a racket", "sings out of tune", "belts out", "flatly intones")
+		else if (loudness < 0)
+			speechverb = pick("hums", "lullabies", "softly intones")
+		else if (singing == "loud")
+			speechverb = pick("belts out", "roars", "yodels")
+		else
+			speechverb = pick("sings",  "croons", "intones", "warbles")
+		text = "<i>[text]</i>"
+	style = "color:thistle;"
```

Comment:
meant to be indented like this?

Reply:
> meant to be indented like this?

nope, my bad

> seems like this would be better done with a prepend of the var quote instead of an override?

not sure what you meant by this comment?